content/4_Gold/DSU.mdx

```diff
@@ -86,106 +85,146 @@ $x$ otherwise.
 </Resources>
 
 ```cpp
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
no need to include this; its just a dsu template

Reply:
i feel like all code snippets should be standalone runnable, but if you really want to

Reply:
I think this doesn't really matter either way so I'll just close and merge